src/components/Integrations/Shopify/Shopify.js

```diff
@@ -1,22 +1,28 @@
-import React, { useState, useEffect } from 'react';
+import React, { useState } from 'react';
 import { Helmet } from 'react-helmet';
 import Loading from '../../Loading/Loading';
 import { InjectAppServices } from '../../../services/pure-di';
 import logo from './logo.svg';
 import { FormattedHTMLMessage, injectIntl, FormattedDate } from 'react-intl';
 import styled from 'styled-components';
 import { SubscriberListState } from '../../../services/shopify-client';
+import { useInterval } from '../../../utils';
 
 const Shopify = ({ intl, dependencies: { shopifyClient } }) => {
-  const [shops, setShops] = useState([]);
-  const [isConnected, setIsConnected] = useState(null);
-  const [error, setError] = useState(null);
-  const [isLoading, setIsLoading] = useState(true);
+  const [shopifyState, setShopifyState] = useState({
+    shops: [],
+    isConnected: false,
+    isConnecting: false,
+    error: null,
+    isLoading: true,
+  });
```

Comment:
I think that this could be enough for default value:

```diff
  const [shopifyState, setShopifyState] = useState({
-    shops: [],
-    isConnected: false,
-    isConnecting: false,
-    error: null,
    isLoading: true,
  });
```